modules/cnv-add-disk-to-vm.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+// ifeval::["{context}" == "cnv-edit-vms"]
+// :cnv-vm-add-disk:
+// :object: virtual machine
+// :object-gui: Virtual Machines
+// endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-add-disk-template:
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-disk_{context}"]
+
+= Adding a virtual disk to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select your {object}.
+. Select the *Disks* tab.
+. Click *Add Disks* to open the *Add Disk* window.
+. In the *Add Disk* window, specify *Source*, *Name*, *Size*, *Interface*, and *Storage Class*.
+. Use the drop-down lists and check boxes to edit the disk configuration.
+. Click *OK*.
+
+// Scrubbing all conditionals used in module
+
+// :object!:
+// :object-gui!:
+
+// ifeval::["{context}" == "cnv-edit-vms"]
+// :cnv-vm-add-disk!:
+// endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-template!:
```

Comment:
We can get rid of this too.
It also doesn't match what was declared earlier (and we're getting rid of that anyway)